CMakeLists.txt

```diff
@@ -133,7 +133,7 @@ set(DLITE_TEMPLATE_DIRS              "share/dlite/templates")
 set(DLITE_STORAGE_PLUGIN_DIRS        "share/dlite/storage-plugins")
 set(DLITE_MAPPING_PLUGIN_DIRS        "share/dlite/mapping-plugins")
 set(DLITE_PYTHON_MAPPING_PLUGIN_DIRS "share/dlite/python-mapping-plugins")
-set(DLITE_PYTHON_STORAGE_PLUGIN_DIRS "share/dlite/python-storage-plugins")
+set(DLITE_PYTHON_STORAGE_PLUGIN_DIRS "share/dlite/python_storage_plugins")
```

Comment:
Changing one directory path to underscore instead of hyphen looks strange. 

For Python 3.5+ you can use the following to import a given file as a python module. This also avoids messing with sys.path
```python
import importlib.util
spec = importlib.util.spec_from_file_location("module.name", "/path/to/file.py")
mod = importlib.util.module_from_spec(spec)
spec.loader.exec_module(mod)
mod.MyClass()
```